tidb-cloud/sql-proxy-account.md

```diff
@@ -0,0 +1,74 @@
+---
+title: SQL Proxy Account
+summary: Learn about the SQL proxy account in TiDB Cloud.
+---
+
+# SQL Proxy Account
+
+A SQL proxy account is a SQL user account that is automatically created by TiDB Cloud for each <!--to be confirmed --> TiDB Cloud user. For example, `testuser@pingcap.com` is a TiDB Cloud user account, while `3jhEcSimm7keKP8.testuser._41mqK6H4` is its corresponding SQL proxy account.
```

Comment:
@zhoubasten It seems that the "each TiDB Cloud user" here is not accurate enough?

Reply:
A SQL proxy account is a SQL user account that is automatically created by TiDB Cloud to access the database by SQL editor and Data Service on behalf of a TiDB Cloud user

Reply:
Updated via https://github.com/pingcap/docs/pull/18696/commits/072970302c23a57e6a01759bb06407f1f0719ae7.